LiteFarmOrg/LiteFarm
# LF-2567: various issues on farm info page

Ticket: [LF-2567](https://lite-farm.atlassian.net/browse/LF-2567)

To test: 
- Go to farm info page
- Enter a farm name longer than 255 characters
- Enter a phone number longer than 20 characters
- Should get inline error for both
